src/HotChocolate/MongoDb/test/Data.MongoDb.Integration.Tests/HotChocolate.Data.MongoDb.Integration.Tests.csproj

```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyName>HotChocolate.Data.MongoDb.Integration.Tests</AssemblyName>
+
+    <!-- Outputs the generated code to the bin directory.  -->
+    <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
+    <CompilerGeneratedFilesOutputPath>$(BaseIntermediateOutputPath)\GeneratedFiles</CompilerGeneratedFilesOutputPath>
+
+    <!-- The directory where all the analyzer bin files are for development time. -->
+    <HotChocolate_BinDir>$(MSBuildThisFileDirectory)\..\..\..\Analyzers\src\Analyzers\bin\Debug\netstandard2.0</HotChocolate_BinDir>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Data\HotChocolate.Data.MongoDb.csproj" />
+    <ProjectReference Include="..\..\..\Core\src\Abstractions\HotChocolate.Abstractions.csproj" />
+    <ProjectReference Include="..\..\..\Core\src\Types\HotChocolate.Types.csproj" />
+    <ProjectReference Include="..\..\..\Analyzers\src\Analyzers\HotChocolate.Analyzers.csproj" ReferenceOutputAssembly="false" OutputItemType="Analyzer" SetTargetFramework="TargetFramework=netstandard2.0" />
```

Comment:
Technically not yet, I left them there because I think we should add a schema first based integration test for MongoDb similar to the existing one in Neo4J. I'll remove for now.